rest-tests/src/test/java/io/syndesis/qe/RestTestHooks.java

```diff
@@ -37,6 +40,15 @@ public void afterTest() {
         stepStorage.flushStepDefinitions();
         log.debug("Flushed steps from steps storage");
         SampleDbConnectionManager.closeConnections();
+
+        if (TestConfiguration.isDeloreanEnvironment()) {
+            //delete all integrations after the test. Only for Delorean since it doesn't support TEST_SUPPORT env for the clean application step
```

Comment:
don't you also need to delete the connections at least?

Reply:
correct. I forget to connections